templates/github/.github/workflows/create-branch.yml.j2

```diff
@@ -60,11 +60,11 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: pulp/plugin_template
-          path: plugin_template
+          path: ../plugin_template
```

Comment:
Checkout will not let you.
I know this is crazy, but you may need to move both checkouts one directory down.

Reply:
Instead you need to add `path: {{ plugin_name }}` to the other checkout.
And now we need to carefully look that all the other scripts still run properly.